python/raft/raft/dask/common/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+# =============================================================================
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+# =============================================================================
+
+include(${raft-python_SOURCE_DIR}/cmake/thirdparty/get_nccl.cmake)
+find_package(ucx REQUIRED)
```

Comment:
I think we still need the `get_ucx.cmake` back in `cpp/cmake/thirdparty` for this to work right? I'm kind of surprised it didn't fail, otherwise.

Reply:
We discussed this offline, but just to close the loop [UCX now ships with a config file](https://github.com/openucx/ucx/commits/master/cmake/ucx-config.cmake.in) so I was able to get rid of this..